src/main/java/duck/Duke.java

```diff
@@ -0,0 +1,142 @@
+package duck;
+
+import duck.operation.CommandGui;
+import duck.operation.Gui;
+import duck.operation.Parser;
+import duck.operation.Storage;
+import duck.task.TaskList;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class Duke extends Application{
```

Comment:
Would it be more consistent to rename your Duke class to Duck just to be more consistent with your project name?